mix.exs

```diff
@@ -1,27 +1,63 @@
-defmodule AlchemistExchange.MixProject do
+defmodule Exchange.MixProject do
   use Mix.Project
 
   def project do
     [
-      apps_path: "apps",
+      app: :exchange,
       version: "0.1.0",
+      build_path: "_build",
+      config_path: "config/config.exs",
+      deps_path: "deps",
+      lockfile: "mix.lock",
+      elixir: "~> 1.10",
       start_permanent: Mix.env() == :prod,
       deps: deps(),
-      test_coverage: [tool: ExCoveralls]
+      name: "Exchange",
+      source_url: "https://github.com/realyarilabs/alchemist_exchange",
+      test_coverage: [tool: ExCoveralls],
+      preferred_cli_env: [
+        coveralls: :test,
+        "coveralls.detail": :test,
+        "coveralls.post": :test,
+        "coveralls.html": :test
+      ]
```

Comment:
need to add `description: description()` and `package: package()` so we can deploy to hex.pm